runtime-modules/content/src/errors.rs

```diff
@@ -205,6 +205,9 @@ decl_error! {
         /// Auction whitelist has only one member
         WhitelistHasOnlyOneMember,
 
+        /// Fail to claim English auction by a non winner
+        MemberClaimingIsNotWinner,
```

Comment:
This should be removed.

Reply:
removed here : 766a142670f3113a33469f205a5a246205951687